src/content/docs/en/guides/images.mdx

```diff
@@ -145,6 +142,22 @@ When using local images in their original aspect ratio, the `width` and `height`
 
 However, both of these properties are required for remote images and images stored in your `public/` folder as Astro is unable to analyze these files.
 
+##### densities
+
+<Since v="3.3.0" /> <Badge>Experimental</Badge>
+
+A list of pixel densities to generate for the image. By default, no other densities are generated. Densities that are equals to widths larger than the original image will be ignored, as to avoid upscaling the image.
+
+This value will be used to generate a `srcset` attribute on the `<img>` tag.
+
+##### widths
+
+<Since v="3.3.0" /> <Badge>Experimental</Badge>
+
+A list of widths to generate for the image. By default, no other widths are generated. Widths that are larger than the original image will be ignored in order to avoid upscaling the image.
+
+This value will be used to generate a `srcset` attribute on the `<img>` tag. When using `widths`, a [`sizes` property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) is also required for `srcset` to take effect.
```

Comment:
``````suggestion
##### widths

<Since v="3.3.0" /> <Badge>Experimental</Badge>

A list of widths to generate for the image. 

If provided, this value will be used to generate a `srcset` attribute on the `<img>` tag. A [`sizes` property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) must also be provided.

Do not provide a value for `densities` when using this value. Only one of these two values can be used to generate a `srcset`.

Widths that are larger than the original image will be ignored to avoid upscaling the image.

```astro
---
import { Image } from 'astro:assets';
import myImage from "../assets/my_image.png";
---
<Image src={myImage} widths={[300, 600, 900]} alt="A description of my image." />
```
``````
Similarly, I think an astro/html example would be nice here!